pycyphal/dsdl/_builtin_form.py

```diff
@@ -48,6 +48,9 @@ def to_builtin(obj: object) -> typing.Dict[str, typing.Any]:
     return out
 
 
+ascii_set = set(string.printable)
```

Comment:
https://peps.python.org/pep-0008/#constants
https://peps.python.org/pep-0008/#descriptive-naming-styles

```suggestion
_ASCII_SET = set(string.printable)
```

Reply:
Disregard this; just remove it.